docs_guide/intro/new_doc_project.rst

```diff
@@ -351,17 +351,22 @@ the `Torch Connector tutorial <https://github.com/Qiskit/qiskit-machine-learning
 the Python package ``torchvision`` to load a dataset. This repo also includes some non-Python dependencies like ``pandoc`` or ``graphviz`` that can't be installed with ``pip``, so
 `apt-get <https://linux.die.net/man/8/apt-get>`_ has to be used instead.
 
-Build and publish to qiskit.org
-"""""""""""""""""""""""""""""""
+Build and publish to qiskit.org/ecosystem
+"""""""""""""""""""""""""""""""""""""""""
 
-If you are deploying your documentation to a qiskit.org domain, then your documentation will need to be deployed to `IBM Cloud Object Storage <https://www.ibm.com/cloud/object-storage>`_, where qiskit.org is hosted, with `Rclone <https://rclone.org>`_, a command-line program that enables you to manage content from the cloud.
+If you are deploying your documentation to a qiskit.org/ecosystem domain, then your documentation will need to be deployed to `IBM Cloud Object Storage <https://www.ibm.com/cloud/object-storage>`_, where qiskit.org is hosted, with `Rclone <https://rclone.org>`_, a command-line program that enables you to manage content from the cloud.
 
-Before starting with this step, it is necessary to ask a `Qiskit organization administrator <https://github.com/orgs/Qiskit/people?query=role%3Aowner>`_ to create a data container or `bucket <https://dataplatform.cloud.ibm.com/docs/content/wsj/manage-data/cos_buckets.html?locale=en>`_ called ``qiskit-org-web-resources/documentation/x``, from whose content the page ``qiskit.org/documentation/x`` will be generated,
+Before starting with this step, it is necessary to ask a `Qiskit organization administrator <https://github.com/orgs/Qiskit/people?query=role%3Aowner>`_ to create a data container or `bucket <https://dataplatform.cloud.ibm.com/docs/content/wsj/manage-data/cos_buckets.html?locale=en>`_ called ``qiskit-org-web-resources/ecosystem/hal-provider``, from whose content the page ``qiskit.org/ecosystem/hal-provider`` will be generated,
```

Comment:
SHould it always be `hal-provider`? Or is that only an example?

Reply:
It's only an example that we use across all of this tutorial. 

In fact `hal-provider` doesn't exist. We literally took the letters of IBM and picked the previous ones (I->H, B->A M->L, so IBM->HAL)

Reply:
Added explicit references to the example in [e8823bb](https://github.com/Qiskit/qiskit_sphinx_theme/pull/270/commits/e8823bbe411b8f35ba10cea47b9ff4e747976630)